mediaapi/storage/postgres/storage.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2017 Vector Creations Ltd
```

Comment:
The current copyright should be `// Copyright 2020 The Matrix.org Foundation C.I.C.`. The way we do it in Synapse is that every new file gets that copyright, and a file that just got moved or changed has it added under the other mentions (e.g. a random file in Synapse I have open right now has:
```
# Copyright 2014-2016 OpenMarket Ltd
# Copyright 2017-2018 New Vector Ltd
# Copyright 2019 The Matrix.org Foundation C.I.C.
```
)

It would be nice to update the copyrights on the files you're changing (I'm only mentioning it on that file but I think it applies across the whole PR).